src/libstd/path.rs

```diff
@@ -296,15 +295,21 @@ impl<'a> Prefix<'a> {
         }
         match *self {
             Verbatim(x) => 4 + os_str_len(x),
-            VerbatimUNC(x,y) => 8 + os_str_len(x) +
-                if os_str_len(y) > 0 { 1 + os_str_len(y) }
-                else { 0 },
+            VerbatimUNC(x, y) => 8 + os_str_len(x) +
```

Comment:
Ughh, rustfmt doesn't do inline if-else expressions well. I'd personally refactor this so that it'd use an intermediate value.